index.html

```diff
@@ -12,34 +12,36 @@
     </head>
     <body>
       <div id="ui">
-	<div id="header">
-	  <div id="title"><h3>Pitt Open Data Project</h3></div>
-	  <div id="navigation">
-	    <ul>
-	      <li><a href="index.html">Home</a></li>
-	      <li><a href="about.html">About</a></li>
-	    </ul>
-	  </div>
-	</div>
-	<div id="sidebar">
-	  <div id="controls">
-	    <h3>Date Range</h3>
-	    <form name="dateSelection">
-	      <div class="dateRadio">
-		<input type="radio" name="radioDate" id="radioMonth" checked>
-		<label for="radioMonth">Past 30 Days</label>
-	      </div>
-	      <div class="dateRadio">
-		<input type="radio" name="radioDate" id="radioWeek">
-		<label for="radioWeek">Past 7 Days</label>
-	      </div>
-	      <div class = "dateRadio">
-		<input type="radio" name="radioDate" id="radioDay">
-		<label for="radioDay">Today</label>
-	      </div>
-	    </form>
-	  </div>
-	</div>
+      	<div id="header">
+      	  <div id="title"><h3>Pitt Open Data Project</h3></div>
+      	  <div id="navigation">
+      	    <ul>
+      	      <li><a href="index.html">Home</a></li>
+      	      <li><a href="about.html">About</a></li>
+      	    </ul>
+      	  </div>
+      	</div>
+        <div id="mobileMenuIcon"></div>
```

Comment:
<img width="869" alt="screen shot 2017-01-25 at 1 38 17 pm" src="https://cloud.githubusercontent.com/assets/4230591/22303973/93e6d392-e303-11e6-9e15-82903fd27634.png">
Safari error console is showing this after clicking on the hamburger button